android/src/main/java/leesiongchan/reactnativeescpos/LayoutBuilder.java

```diff
@@ -88,6 +92,29 @@ public String createAccent(String text, char accent, int charsOnLine) {
         return createTextOnLine(' ' + text + ' ', accent, TEXT_ALIGNMENT_CENTER, charsOnLine);
     }
 
+    public String createCustomNumberOfString(String text) {
+        String symbol = "", count = "0";
+
+        if (text.contains("{RP:")) {
+            symbol = text.substring(0,text.indexOf("{RP:"));
+            count = text.substring(text.indexOf("{RP:") + 4, text.indexOf('}'));
+        }
+
+        if (text.contains("{C}")) {
```

Comment:
Do we need this? I think {C} and {R} already been handled

Reply:
Let me test and get back to you.
By logically it doesn't work

Reply:
yes we need. u must include {C} {R}